backend/app/main/views.py

```diff
@@ -204,40 +228,19 @@ def get_a_client_from_email(email):
         return "There is no client with that email. Please try again.", 422
 
     # Create and return object
-    c = Client(name=name, email=email, id_number=id_number,
-               notes=notes, attachments=attachments)
+    c = Client(
+        name=name,
+        email=email,
+        id_number=id_number,
+        notes=notes,
+        attachments=attachments,
+    )
     return jsonify(c.serialize()), 200
 
 
-def handleEmailResponse(res):
```

Comment:
how come these were removed?

Reply:
I just moved them to the bottom of the file so that the grouping was more logical

Reply:
nvm I see they were moved to helpers below